app/celery/scheduled_tasks.py

```diff
@@ -302,3 +303,15 @@ def populate_monthly_billing():
     start_date, end_date = get_month_start_and_end_date_in_utc(yesterday_in_bst)
     services = get_service_ids_that_need_billing_populated(start_date=start_date, end_date=end_date)
     [create_or_update_monthly_billing(service_id=s.service_id, billing_month=end_date) for s in services]
+
+
+@notify_celery.task(name="run-letter-jobs")
+@statsd(namespace="tasks")
+def run_letter_jobs():
+    ids = dao_get_letter_jobs_by_status(JOB_STATUS_READY_TO_SEND)
```

Comment:
Being really nit picky, worth renaming to `job_ids` just so it's clear what we're passing to `send_task`